lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -234,14 +234,16 @@ void LocalEnforcer::aggregate_records(
   // Insert the IMSI+SessionID for sessions we received a rule record into a set
   // for easy access
   std::unordered_set<ImsiAndSessionID> sessions_with_reporting_flows;
-  for (const RuleRecord& record : records.records()) {
+  // In some failure cases, PipelineD may still hold onto flows for sessions
+  // that do not exist in SessionD. In this case, send DeactivateFlowsRequest
+  std::vector<RuleRecord> dead_sessions_to_cleanup;
+  for (const RuleRecord record : records.records()) {
     const std::string &imsi = record.sid(), &ip = record.ue_ipv4();
     // TODO IPv6 add ipv6 to search criteria
     SessionSearchCriteria criteria(imsi, IMSI_AND_UE_IPV4_OR_IPV6, ip);
     auto session_it = session_store_.find_session(session_map, criteria);
     if (!session_it) {
-      MLOG(MERROR) << "Could not find session for " << imsi << " and " << ip
-                   << " during record aggregation";
+      dead_sessions_to_cleanup.push_back(record);
```

Comment:
did you see that case?
I am asking because i wasn't able to ever see it in teravm
But it make sense add this here

Reply:
@themarwhal Yeah seeing a lot in v1.4. I expect to start seeing it in master also because we start terminating sessions that cannot install flows properly.

Reply:
(meaning, that if the grpc calls to activate flows are failing, the deactivate flows calls when terminating might fail too.)

Reply:
ohh right. In teravm we haven't been able to run load tests like you are doing. That is why.